src/components/settings/Encounter.vue

```diff
@@ -86,148 +94,216 @@
 </template>
 
 <script>
-	import { mapGetters, mapActions } from "vuex";
+import { mapGetters, mapActions } from "vuex";
+
+export default {
+	name: "EncounterSettings",
+	data() {
+		return {
+			userId: this.$store.getters.user.uid,
+			types: {
+				general: {
+					type_settings: [
+						{
+							key: "automate",
+							name: "Automate",
+							icon: "fas fa-calculator",
+							info:
+								"<p>When you check the automate setting, some actions in an encounter will be done automatically.<br/>" +
+								"At the moment only one action is dependent on the automate function.</p>" +
+								"<b>Death Saving Throws</b>" +
+								"<p>When a down player receives damage, it will automatically get a failed death saving throw." +
+								'If the "critical hit" checkbox is checked, two death saves will be failed.<br/>' +
+								"Even a character that is stable at 0 hit points will automatically fail saves when taking damage.</p>",
+							options: [
+								{
+									value: false,
+									name: "Don't automate",
+									action: "Don't",
+									icon: "fas fa-times",
+									color: "red",
+								},
+								{
+									value: undefined,
+									name: "Automate",
+									action: "Automate",
+									icon: "fas fa-check",
+									color: "green",
+								},
+							],
+						},
+						{
+							key: "background",
+							name: "Background Image",
+							icon: "fas fa-image",
+							info: "Show or hide background image you set for the encounter.",
+							options: [
+								{
+									value: undefined,
+									name: "Hidden",
+									action: "Hide",
+									icon: "fas fa-eye-slash",
+									color: "red",
+								},
+								{ value: true, name: "Shown", action: "Show", icon: "fas fa-eye", color: "green" },
+							],
+						},
+						{
+							key: "initOrder",
+							name: "Initiative order",
+							icon: "fas fa-arrows-alt-v",
+							info: "Change the order of initiative.",
+							options: [
+								{
+									value: undefined,
+									name: "Descend",
+									action: "Descend",
+									icon: "fas fa-long-arrow-alt-down",
+									color: "neutral-2",
+								},
+								{
+									value: true,
+									name: "Ascend",
+									action: "Ascend",
+									icon: "fas fa-long-arrow-alt-up",
+									color: "neutral-2",
+								},
+							],
+						},
+						{
+							key: "critical",
+							name: "Critical Hits",
+							icon: "far fa-crosshairs",
+							info: `How do you want critical hits to be handled?<br/>
+                <b>Roll</b>: all the damage dice are rolled twice.<br/>
+                <b>Double</b>: the rolled damage is doubled.
+                <b>Max+Roll</b>: the max damage is added to the rolled damage.`,
```

Comment:
Het is lastig precies goed te omschrijven hoe het werkt
```suggestion
                <b>Max+Roll</b>:  the max value of the dice is added to the rolled damage.`,
```